src/services/getConcertsOfCity.js

```diff
@@ -0,0 +1,11 @@
+import getConcertDetails from './getConcertDetails'
+
+export default function getConcertsOfCity(currentCity) {
+  const url =
+    'https://app.ticketmaster.com/discovery/v2/events?apikey=7elxdku9GGG5k8j0Xm8KWdANDgecHMV0&locale=*&sort=date,asc&countryCode=DE&classificationName=music'
```

Comment:
Please don't add your API Key to the front end! It should be in a `.env` folder or even better in a backend. Here is an article about [env and create-react-app](https://create-react-app.dev/docs/adding-custom-environment-variables/). And you could add an `api` folder. Please ask Mel about it, he uses it. 
```suggestion
    `https://app.ticketmaster.com/discovery/v2/events?apikey=${API_KEY}&locale=*&sort=date,asc&countryCode=DE&classificationName=music`
```

Reply:
done :)